packages/js/src/MatomoTracker.ts

```diff
@@ -24,10 +24,15 @@ class MatomoTracker {
     window._paq = window._paq || []
 
     if (window._paq.length === 0) {
-      window._paq.push(['enableLinkTracking'])
       window._paq.push(['setTrackerUrl', trackerUrl || `${urlBase}matomo.php`])
       window._paq.push(['setSiteId', siteId])
 
+      // accurately measure the time spent on the last pageview of a visit
+      window._paq.push(['enableHeartBeatTimer'])
+      // measure outbound links and downloads
+      // might not work accurately on SPAs
```

Comment:
```suggestion
      // might not work accurately on SPAs because new links (dom elements) are created dynamically without a server-side page reload.
```